output_http.go

```diff
@@ -301,7 +301,12 @@ func (c *HTTPClient) Send(data []byte) ([]byte, error) {
 		req.Host = c.config.url.Host
 	}
 
-	req.URL = c.config.url
+	if c.config.url.Path == "" {
+		req.URL, _ = url.ParseRequestURI(c.config.url.Scheme + "://" + c.config.url.Host + req.RequestURI)
```

Comment:
to avoid unexpected error from `url.ParseRequestURI` can we just do
```suggestion
	if c.config.url.Path == "" && c.config.url.RawQuery == "" {
		req.URL.Scheme = c.config.url.Scheme
		req.URL.Host = c.config.url.Host
```

Reply:
@two any update on this?

Reply:
OK